VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/services/ProviderService.java

```diff
@@ -28,4 +29,30 @@ public Provider persistProvider(Provider provider) throws SQLException {
         provider.setProviderId(providerId);
         return provider;
     }
+
+    public List<Provider> getProvider(String id) throws Exception{
+        long num_id =Long.parseLong(id);
```

Comment:
We follow camelCase for java, underscores are more used for having column names in db